src/components/expression/expression-renderer.js

```diff
@@ -210,6 +203,51 @@ genotet.ExpressionRenderer.prototype.initLayout = function() {
    */
   this.svgConditionLabels_ = this.svgHeatmap_.append('g')
     .classed('condition-labels', true);
+
+  /**
+   * SVG group for the heatmap gradient.
+   * @private {!d3.selection}
+   */
+  this.svgHeatmapGradient_ = this.svgHeatmap_.append('g')
+    .classed('heatmap-gradient', true);
+
+  /**
+   * SVG defs for the heatmap gradient.
+   * @private {!d3.selection}
+   */
+  this.gradientContent_ = this.svgHeatmapGradient_.append('defs')
```

Comment:
But its parent svg is appened in js. We can write like this and load it as template. But I think it is not that necessary...